docs/technical/continuous-deployment-applications.md

```diff
@@ -0,0 +1,33 @@
+# Continuous Deployment Strategy for Applications
+
+This document outlines a high-level strategy and approach for Continuous Deployment pipelines across the whole Bürokratt system.
+
+Continuous Deployment (CD) is the practice of automatically releasing changes from the (`main`) branch frequently and in doing so "continuously" ensuring that the code "integrates" well with other code.
+
+By combining Continuous Integration and infrastructure as code (IaC), we're able to achieve identical deployments with the confidence to be able to deploy to production at any time. CD allows us to automate the entire process from code commit to production if CI/CD tests and security checks pass.
```

Comment:
```suggestion
By combining Continuous Integration and Infrastructure as Code (IaC), we're able to achieve identical deployments with the confidence to be able to deploy to production at any time. CD allows us to automate the entire process from code commit to production if CI/CD tests and security checks pass.
```

Reply:
Are there slightly more words to talk about the concept of 'identical' deployments?  There's a devops principle around immutable build artefacts - and therefore exactly the same code is deployed to all environments -  is that what is being expressed here? (I'm probably being thick here...)